examples/pinot_live.py

```diff
@@ -0,0 +1,50 @@
+from pinotdb import connect
```

Comment:
pinot.live is brought down for a while, we should bring it back :p

Reply:
yeah I will add some CI task later. technically we should download the docker image and spin up a test env. 
will see how to set it up.